nixos/modules/services/misc/workout-tracker.nix

```diff
@@ -0,0 +1,121 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  inherit (lib) types;
+  cfg = config.services.workout-tracker;
+  exe = lib.getExe cfg.package;
+in
+
+{
+  options = {
+    services.workout-tracker = {
+      enable = lib.mkEnableOption "A workout tracking web application for personal use (or family, friends), geared towards running and other GPX-based activities";
+
+      package = lib.mkPackageOption pkgs "workout-tracker" { };
+
+      address = mkOption {
+        type = types.str;
+        default = "localhost";
+        description = lib.mdDoc "Web interface address.";
```

Comment:
You can also remove all `lib.mdDoc` as it has no effect anymore (just use the string),